ci/requirements-3.6-dev.yml

```diff
@@ -20,16 +20,16 @@ dependencies:
   - mypy
   - numpy>=1.11
   - openjdk=8
-  - pandas>=0.21
+  - pandas>=1.0
   - pip
   - plumbum
   - pre-commit
   - psycopg2
-  - pyarrow>=0.12
+  - pyarrow>=0.13
   - pydata-google-auth
   - pydocstyle=4.0.1
   - pygit2
-  - pymapd>=0.12.0
+  - pymapd=0.21.0
```

Comment:
remove this from this build (why did you bump this SO much)??

Reply:
using the previous pinning, conda automatically pinned that to 0.21 so that is why I just used that.